openshift_metrics/invoice.py

```diff
@@ -150,13 +191,7 @@ class ProjectInvoce:
 
     def add_pod(self, pod: Pod) -> None:
         """Aggregate a pods data"""
-        su_type, su_count, _ = Pod.get_service_unit(
-            cpu_count=pod.cpu_request,
-            memory_count=pod.memory_request,
-            gpu_count=pod.gpu_request,
-            gpu_type=pod.gpu_type,
-            gpu_resource=pod.gpu_resource,
-        )
+        su_type, su_count, _ = pod.get_service_unit()
```

Comment:
Isn't `get_service_unit()` returning a `ServiceUnit` object now?

Reply:
@knikolla It was returning a ServiceUnit namedtuple even before this PR. This doesn't change that. Is there anything wrong here?